src/components/Combobox.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div class="combobox">
+    <input
+      type="text"
+      :id="id"
+      class="combobox__input"
+      ref="comboboxInput"
+      :value="value"
+      :placeholder="placeholder"
+      @keydown="checkInput"
+      @input="$emit('input', $event.target.value)"
+      autocomplete="off"
+    />
+    <ul v-if="source" class="combobox__options" ref="comboboxList">
+      <li
+        v-for="(option, index) in source"
+        :key="index"
+        class="combobox__option"
+      >
+        {{ option }}
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script>
+import Combobo from 'combobo';
+
+export default {
+  name: 'Combobox',
+  props: {
+    id: String,
+    source: Array,
+    value: String,
+    allowCustomInput: {
+      type: Boolean,
+      default: true,
+    },
+    placeholder: String,
+  },
+  methods: {
+    init() {
+      const { comboboxInput, comboboxList } = this.$refs;
+
+      this.combobo = new Combobo({
+        input: comboboxInput,
+        list: comboboxList,
+        options: this.optionEls,
+        activeClass: 'combobox__option--selected',
+      });
+
+      this.combobo.addEventListener('selection', () => {
+        this.$nextTick(() => {
+          this.$emit('input', this.$refs.comboboxInput.value);
+        });
+      });
+    },
+    checkInput(e) {
+      const alwaysAllowed = [9, 13, 27, 16, 8]; // ignore tab, enter, escape, shift, backspace
+
+      if (alwaysAllowed.indexOf(e.which) >= 0) {
```

Comment:
[`array.includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) would be more explicit.

Also `indexOf` return `0` if the first element matches, so checking for `>= 0` might not be what you want here.

Reply:
Hm, `array.includes` reads better, but does not work in IE11, not sure if we want to polyfill, I'll double check our browser requirements.

Reply:
Doublechecked, IE11 is not a requirement, `array.includes` it is!